src/lib/lib3index/cholesky.cc

```diff
@@ -1,12 +1,7 @@
 /*
- * @BEGIN LICENSE
+ *@BEGIN LICENSE
```

Comment:
It looks like you merged the files `cholesky.cc`, `cholesky.h`, and `CDJK.cc` incorrectly. Please correct these license templates to the current 2016 template (effectively what you changed here).